<launch>
        <node pkg="color_pc" type="filter_node" name="filter_node" args="true" output="screen" />
	<node pkg="color_pc" type="color_pc_sync_node" name="color_pc_sync_node" args="false" output="screen">
		<rosparam file="$(find color_pc)/config/param2.yaml" command="load"/>	
	</node>
	<node pkg="rviz" type="rviz" name="rviz_color_pc_sync" args="-d $(find color_pc)/config/show.rviz" required="true" />
</launch>
